js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -3126,4 +3126,56 @@ identityBrokeringLink=Identity brokering link
 searchClientRegistration=Search for policy
 importFileHelp=File to import a key
 logo=Logo
-avatarImage=Avatar image
\ No newline at end of file
+avatarImage=Avatar image
+organizations=Organizations
+organizationsList=Organizations
+organizationsExplain=Manage your organizations and members.
+emptyOrganizations=No organizations
+emptyOrganizationsInstructions=There is no organization yet. Please create an organization and manage it.
+searchOrganization=Search for organization
+domains=Domains
+organizationDelete=Delete organization?
+organizationDeleteConfirm=Are you sure you want to permanently delete this organization? If so, all the data of this organization will be deleted.
+organizationDeletedSuccess=The organization has been deleted
+orgainzatinoDeleteError=Could not delete client\: {{error}}
+createOrganization=Create organization
+domain=Domain
+organizationDomainHelp=What is a domain
```

Comment:
I suppose this help text is temporary, right?

Reply:
Yes, I didn't have text to explain this field yet

Reply:
@edewit Pedro has provided a help text for the Domain. FYI: 
"Sets one or more domains to an organization so that users are mapped to an organization whenever their email domains match one of the domains herein set forth."